protos/sources/vega/data/v1/data.proto

```diff
@@ -31,6 +31,9 @@ message Property {
   string name = 1;
   // value is the value of the property.
   string value = 2;
+  // An optional decimal place to be be applied on the provided value
+  // valid only for PropertyType of type DECIMAL and INTEGER
+  uint32 number_decimal_places = 3;
```

Comment:
why is this necessary? we already know it from the spec

Reply:
Yes, that decimal will be already applied in the string value.

Reply:
This message is deserialized into this object from the datanode side - https://github.com/vegaprotocol/vega/blob/develop/datanode/entities/data.go#L32